admin/orders.php

```diff
@@ -1270,10 +1270,10 @@ function couponpopupWindow(url) {
                     }
 
                     $show_difference = '';
-                    if ((strtoupper($orders->fields['delivery_name']) !== strtoupper($orders->fields['billing_name']) && trim($orders->fields['delivery_name']) !== '')) {
+                    if (trim($orders->fields['delivery_name']) !== '' && (strtoupper($orders->fields['delivery_name']) !== strtoupper($orders->fields['billing_name']))) {
```

Comment:
When rewriting with this swap, it now seems like switching out `trim() !== ''` with `!empty()` would be sensible, on both lines.